snuba/datasets/discover.py

```diff
@@ -264,7 +282,24 @@ def get_query_processors(self) -> Sequence[QueryProcessor]:
         discover_source = self.get_dataset_schemas().get_read_schema().get_data_source()
 
         return [
-            DatasetSelector(discover_source, self.__events_columns, self.__transactions_columns),
+            DatasetSelector(
+                discover_source=discover_source,
+                events_columns=self.__events_columns,
+                transactions_columns=self.__transactions_columns,
+                post_processors={
+                    TRANSACTIONS: [
+                        NestedFieldConditionOptimizer(
+                            "tags", "_tags_flattened", "timestamp", BEGINNING_OF_TIME
```

Comment:
Can we safely use `timestamp` here or does it need to be the actual column name on the transactions table?

Reply:
That's the column we receive on the query. Timestamp is resolved after this processor so we are fine.

Reply:
Actually I should change this to support multiple columns since the condition can be on timestamp, start_ts and finish_ts

Reply:
done